lib/audited/auditor.rb

```diff
@@ -33,7 +33,7 @@ module ClassMethods
       #       audited except: :password
       #     end
       #
-      # * +require_comment+ - Ensures that audit_comment is supplied before
+      # * +comment_required+ - Ensures that audit_comment is supplied before
```

Comment:
Kind of related documentation fix: The private method is called `require_comment`, the option is called `comment_required`.